website/docs/cli/install/apt.mdx

```diff
@@ -58,12 +58,10 @@ sudo apt install terraform
 
 ## Supported Architectures
 
-The HashiCorp APT server currently has packages only for the `amd64`
-architecture, which is also sometimes known as `x86_64`.
-
-There are no official packages available for other architectures, such as
-`arm64`. If you wish to use Terraform on a non-`amd64` system,
-[download a normal release `.zip` file](/downloads) instead.
+You can [download a `.zip` file](/downloads) from the The HashiCorp Yum/DNF server for the following packages:
```

Comment:
This page is about the APT server, not Yum.

Also, you cannot download a `.zip` file from the APT or Yum server. I think this is a semantic mistake in the editing here. You can download a `.deb` from APT or a `.rpm` from Yum.

After this edit we are also now missing the original fallback instructions to use the downloads page to retrieve a `.zip`.